src/main/java/jenkins/plugins/slack/workflow/SlackSendStep.java

```diff
@@ -144,11 +144,14 @@ public String getAttachments(){
         return attachments;
     }
 
-    @DataBoundConstructor
-    public SlackSendStep(@Nonnull String message) {
+    @DataBoundSetter
+    public void setMessage(String message) {
         this.message = message;
     }
 
+    @DataBoundConstructor
+    public SlackSendStep() { }
```

Comment:
We could keep the old ctor but I doubt anyone is calling the constructor except for this plugin.

Reply:
happy to remove

Reply:
do you need this constructor?

Reply:
We need to keep a `@DataBoundConstructor` otherwise stapler is not happy 😅
I was thinking the old ctor with `SlackSendStep(String message)`